apps/mobile/src/components/DebugScreen/components/LanguagePicker.tsx

```diff
@@ -31,11 +30,7 @@ function LanguagePicker(): JSX.Element {
       </Picker>
       <Button
         onPress={() => {
-          const newI18n = new I18n(translations)
```

Comment:
Are you sure? Current app language is just the preferences atom it won't modify I18n settings